lupa/tests/test.py

```diff
@@ -2619,6 +2619,119 @@ def test_nil_debug_traceback(self):
         except lupa.LuaError as e:
             self.assertNotIn("stack traceback:", e.args[0])
 
+################################################################################
+# tests for handling overflow
+
+class TestOverflow(SetupLuaRuntimeMixin):
+    maxinteger = sys.maxsize # maximum value for C Py_ssize_t
+    biginteger = (maxinteger + 1) << 1 # value too big to fit in C size_t
+    maxfloat = sys.float_info.max # maximum value for Python float
+    bigfloat = int(maxfloat) * 2 # value too big to fit in Python float
```

Comment:
PEP-8 wants two spaces before the comment, but for the sake of readability, let's just align them.
```suggestion
    maxinteger = sys.maxsize            # maximum value for C Py_ssize_t
    biginteger = (maxinteger + 1) << 1  # value too big to fit in C size_t
    maxfloat = sys.float_info.max       # maximum value for Python float
    bigfloat = int(maxfloat) * 2        # value too big to fit in Python float
```